spec.html

```diff
@@ -21329,6 +21329,12 @@ <h1>Static Semantics: Early Errors</h1>
             It is a Syntax Error if |Initializer| is not present and IsConstantDeclaration of the |LexicalDeclaration| containing this |LexicalBinding| is *true*.
           </li>
         </ul>
+        <emu-grammar>LexicalBinding : BindingPattern Initializer</emu-grammar>
+        <ul>
+          <li>
+            It is a Syntax Error IsUsingDeclaration of the |LexicalDeclaration| containing this |LexicalBinding| is *true*.
```

Comment:
```suggestion
            It is a Syntax Error if IsUsingDeclaration of the |LexicalDeclaration| containing this |LexicalBinding| is *true*.
```

Reply:
That's what I get for having to transcribe this in three different places.